asapdiscovery-data/asapdiscovery/data/openeye.py

```diff
@@ -416,186 +417,6 @@ def openeye_perceive_residues(prot: oechem.OEGraphMol) -> oechem.OEGraphMol:
     return prot
 
 
-def split_openeye_mol(complex_mol, lig_chain="A", prot_cutoff_len=10):
```

Comment:
Yep i think this is a good idea to move this out of the core `openeye` functionality, just make sure to change the call sites as you have already started doing. :+1:

Reply:
I'm using PyCharm to "refactor" the function and move it, which automatically moves all the imports, and I'm also double checking with checking for usage. Hopefully I'll catch everything!